packages/wallets/solflare/src/adapter.ts

```diff
@@ -71,9 +71,8 @@ export class SolflareWalletAdapter extends BaseMessageSignerWalletAdapter {
                 }
                 return false;
             });
+            scopePollingDetectionStrategy(detectAndRegisterSolflareMetaMaskWallet);
```

Comment:
We want to poll to detect rather than detect one time. This will increase the robustness of the detection if the extension doesn't inject fast.